.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
   "cSpell.enabled": true,
   "editor.defaultFormatter": "dbaeumer.vscode-eslint",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
would be better to do a separate pr for lint/format changes

Reply:
accidental change slipped in. reverted: 3475e167aacb79718bf4b992933cb0748739b679